.github/workflows/release.yml

```diff
@@ -5,6 +5,11 @@ on:
     tags:
       - v**
 
+  workflow_run:
+    workflows: ["Automatically release a new version of flyctl"]
```

Comment:
I think this might need `types`, like this:
```suggestion
    workflows: ["Automatically release a new version of flyctl"]
    types:
      - completed
```

Reply:
We might also want to limit this to the main branch?

Reply:
hm, i think that the tag workflow only runs on the main branch, so it should be fine?

Reply:
or at least, it should by default i think